ext/bcmath/tests/bcceil.phpt

```diff
@@ -0,0 +1,95 @@
+--TEST--
+bcceil() function
+--EXTENSIONS--
+bcmath
+--FILE--
+<?php
+$nums = [
+    '0',
+    '0.00',
+    '-0',
+    '-0.00',
+    '0.01',
+    '0.000000000000000000000000000000000000000001',
+    '-0.01',
+    '-0.000000000000000000000000000000000000000001',
+    '1',
+    '1.0000',
+    '1.0001',
+    '100000.000000000000000000000000000000000000000001',
+    '-1',
+    '-1.0000',
+    '-1.0001',
+    '-100000.000000000000000000000000000000000000000001',
+];
+
+foreach ($nums as $num) {
+    echo $num."\n";
+    echo '=> '.bcceil($num)."\n\n";
+}
+
+echo "========== value error ==========\n";
+try {
+    bcceil('hoge');
+} catch (Throwable $e) {
+    echo $e->getMessage()."\n";
+}
+
+try {
+    bcceil('0.00.1');
+} catch (Throwable $e) {
+    echo $e->getMessage()."\n";
+}
```

Comment:
I prefer to split the error cases into a dedicated test, as the error messages might change (happened multiple times, although hopefully it won't again)